### DISCUSSION TITLE
After install the a new dashboard using `php artisan chimera:install` then running `npm install` to install the npm packages, when we run `npm run dev` there is a misisng library "lodash" therefore an error occurs, we have to manually install lodash to continue.

### DISCUSSION THREAD

**Discussion by baakal** [tech-acs/chimera-docs#10] [Bug report] [answered]
**version:** 2.7.1
**Steps to reproduce:**

1. `composer create-project laravel/laravel dashboard` then `cd dashboard`
2. `composer require uneca/dashboard-starter-kit`
3. `php artisan chimera:install`
4. create database and update .env
5. `php artisan migrate`
6. `npm install`
7. `npm run dev`

Error produced:

```bash
The following dependencies are imported but could not be resolved:

  lodash/isUndefined (imported by **********************************************/dashboard/resources/js/LeafletMap.js)
  lodash/pick (imported by imported by **********************************************/dashboard/resources/js/PlotlyChart.js)
  lodash/isNull (imported by imported by **********************************************/dashboard/resources/js/LeafletMap.js)
  lodash/isEmpty (imported by imported by **********************************************/dashboard/resources/js/LeafletMap.js)
  lodash/keyBy (imported by imported by **********************************************/dashboard/resources/js/LeafletMap.js)

Are they installed?
```

**Expected behavior** is the  `php artisan chimera:install` should include lodash on package.json.

Workaround: run the following command:
 `npm install lodash --save`

**Comment by amestsantim** [comment, marked as answer]
The cause for this error is that lodash has just recently been removed from the laravel codebase.
https://github.com/laravel/laravel/pull/6095

This has been rectified in release 2.7.3
https://github.com/tech-acs/chimera-starter-kit/releases/tag/2.7.3
